docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -32,6 +32,14 @@ Here are some examples of customizing the component. You can learn more about th
 ## Positioned snackbars
 
 There may be circumstances when the placement of the snackbar needs to be more flexible.
+You can control the position of the snackbar by specifying the `anchorOrigin` prop, with the following interface:
```

Comment:
Are we in danger of reproducing the API docs in the demo pages? It would get a bit unweildy if we did that for every prop...

I'd suggest just: 
```suggestion
You can control the position of the snackbar by specifying the `anchorOrigin` prop.
```

Reply:
Fair enough